src/seedu/addressbook/data/person/Address.java

```diff
@@ -36,24 +33,4 @@ public static boolean isValidAddress(String test) {
         return test.matches(ADDRESS_VALIDATION_REGEX);
     }
 
-    @Override
-    public String toString() {
-        return value;
-    }
-
-    @Override
-    public boolean equals(Object other) {
```

Comment:
you need to override `Contact.equals` in `Address` so that you you can check equality to other Address objects